packages/ai/core/generate-transcript/index.ts

```diff
@@ -0,0 +1,12 @@
+export { generateTranscript as experimental_generateTranscript } from './generate-transcript';
+export type { GenerateTranscriptResult as Experimental_GenerateTranscriptResult } from './generate-transcript-result';
+export type GeneratedTranscript = {
+  text: string;
+  segments: Array<{
+    text: string;
+    start: number;
+    end: number;
+  }>;
+  language: string | undefined;
+  duration: number | undefined;
+};
```

Comment:
What is this used for? How is it related to `GenerateTranscriptResult`?

(we don't usually define types in index files; they are used for exporting only)